.drone.yml

```
@@ -0,0 +1,89 @@
+clone:
+  git:
+    image: registry.fpfis.eu/drone-plugins/git:next
+    when:
+      event: [ push, pull_request, tag ]
+  git:
+    image: registry.fpfis.eu/drone-plugins/git
+    when:
+      event: [ deployment ]
+
+workspace:
+  base: /test
+  path: oe_content_extra
+
+services:
+  web:
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    environment:
+      - DOCUMENT_ROOT=/test/oe_content_extra
+      - COMPOSERVER=--2
+  mysql:
+    image: registry.fpfis.eu/fpfis/sql:percona-5.7
+    command: --innodb-log-file-size=1G --max_allowed_packet=1G --innodb-buffer-pool-size=512M --wait_timeout=3000 --net_write_timeout=3000 --log_error_verbosity=3
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
```

do we need MYSQL_DATABASE?
```suggestion
      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
      - MYSQL_DATABASE=drupal
```

---

It's not really needed, Drupal will create the db anyway.